dev/README.md

```diff
@@ -1,13 +1,18 @@
 > This README contains expected Markdown elements. It is useful for developing themes and testing that changes to Oranda parsing do not have unexpected repercussions. If you run into a bug not captured here, please add it.
 
 # Heading 1
+
 ## Heading 2
+
 ### Heading 3
+
 #### Heading 4
+
 ###### Heading 5
+
 ####### Heading 6
 
-A paragraph can contain **bold**, _italic_ and *italic* text. (Those two are different.) Let's look at some longer text and other features.
+A paragraph can contain **bold**, _italic_ and _italic_ text. (Those two are different.) Let's look at some longer text and other features.
```

Comment:
I think prettier needs to be turned off here. 😉

Reply:
done!